docs/faq.md

```diff
@@ -34,8 +34,31 @@ Also the `Max Users Per Team` setting in Mattermost System Console should be eno
 
 ### Agent failing with `MaxActiveUsers is not compatible with max Rlimit value` error
 
-This means the maximum number of file descriptors is lower than what the agent needs to operate.  
+This means the maximum number of file descriptors is lower than what the agent needs to operate. 
+
 The following command can be run to raise the limit to the suggested value:
 ```sh
 ulimit -n VALUE
 ```
+
+#### Note
+For terraform deployments, this value is hard coded as 65535 in the `systemd` file for the loadtest api. If you are attempting to test over ~52k users you will need to manually update this file with the instructions below.
+
+1. ssh into your loadtest agents. You can see the agents available by running `go run ./cmd/ltctl ssh`.
+2. Modify the `/lib/systemd/system/ltapi.service` file with the below change.
+
+```diff
++ LimitNOFILE=150000
+- LimitNOFILE=65535
+```
```

Comment:
The instance size of the agent also plays a role here. I'd rather advise to bump up the number of agents horizontally rather than bump it up vertically.

What I'd suggest is to just mention something like:

```
For terraform deployments, this value is hard coded as 65535 in the `systemd` file for the loadtest api. If you need to change the value, you'll have to change the `LimitNOFILE` value in `/lib/systemd/system/ltapi.service` file.
```

Reply:
You're saying that if I wanted to do a load test with 100k users. I simply need to add additional agent nodes to my deployer.json?

Reply:
Let me know if that clarification works. I left the instructions but removed the specifics, just to clear up potential questions. If we still don't want them, i'll drop it to just the one line.

Reply:
> You're saying that if I wanted to do a load test with 100k users. I simply need to add additional agent nodes to my deployer.json?

That is correct. Only on the agent side though. You'll need to similarly scale up the server nodes and the DB nodes. Although you will eventually get bottlenecked on the intra cluster traffic.

Reply:
Interesting. I tried that and it was still yelling at me, but let me give it another go now that I understand this a bit more. Is there a limit to agent nodes on a load test where it stops working?